src/edit_python_pe/github_client.py

```
@@ -0,0 +1,152 @@
+def create_pr(
+    file_content: str,
+    current_file: str | None,
+    repo_path: str,
+    original_repo: Repository,
+    forked_repo: Repository,
+    token: str,
+    aliases: list[str],
+    name: str,
+    email: str,
+) -> tuple[str, str | None]:
+    name_file, file_path = _create_member_file(
```

Ruff (0.15.13)

[warning] Unpacked variable `file_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)